src/index.js

```diff
@@ -0,0 +1,13 @@
+import http from 'http';
+import { router } from './router/index.js';
+import { ConfigService } from './config/ConfigService.js';
+
+const configService = new ConfigService();
```

Comment:
You should create 1 instance, when you export. You shouldn't create every time new instances